.travis.yml

```diff
@@ -38,14 +38,13 @@ before_install:
       # TODO: need script for Windows to generate SSL certs...
       # Install Redis
       choco install redis-64 -v
+      choco install openssl.light
+      mkdir certs
+      ./tools/generate-ssl-certs.sh
       redis-server --service-install
       redis-server --service-start
       redis-cli info
     fi
 script:
   - npm run test-ci
 matrix:
-  allow_failures:
```

Comment:
We can do this, but we'll have to be very aware when builds fail and it's only due to Travis not being able to download redis/openssl.  As long as we educate the rest of the team, it should be fine.